nginx/jwt_public_key.pem

```diff
@@ -0,0 +1,8 @@
+-----BEGIN PUBLIC KEY-----
+MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAoWPv5B06KeQPru1unPs1
+k/iAME1nIEsoWJIkmaP5broZcUpztOpmRAf8+RCk5oAXQrJ87SoIs8UG4fKCduNa
+FXsKvv1zn6z+S92ScpHOYmuxO9mlL1np9NbhRdiSDwIomdpXNZ+ttF+Ll0nO87wU
+aKvTdwLhLswSLEjt2YwmKOhgpSPe2mLkk5Me1v7iAfj4ermisaV/WLu7QbelboZO
+Yp3yhOkPVKhvJU4fBkx/cnkcvLMo8dx5sM3TQonZB5oiv1w4YWz7nBjbmH3bv+9u
+AiLpr9K2ueieZbM1R3f7vhS44pNUI5LYRYA4NHEv1O+n5bE8BuvnVMG5qGP6fB8J
+uwIDAQAB-----END PUBLIC KEY-----
```

Comment:
Should be  generated from an environment variable. For instance you could define the public and private keys in a .env injected in the docker-compose. Then  when the nginx container mounts,  this file is generated with the right content

Reply:
Actually this file is/should be gitingored and the value is copied in nginx.conf in the dockerfile. (I will add jwt_public_key.pem.dist)